### PR TITLE
Novel editor

### PR DESCRIPTION
**Pull request by devodii**
Feat: Presentational Novel eidtor only rendered in puck without dynamic config.

### CODE REVIEW

**Review by BleedingDev — CHANGES_REQUESTED** [review]

File: components/CustomInputs/TextEditorInput.tsx [hunk 1]
```diff
@@ -1,18 +1,29 @@
...
-      <Editor
-        defaultValue={value.raw}
-        completionApi='/'
-        onDebouncedUpdate={(editor) => {
-          onChange({ raw: editor?.storage.markdown.getMarkdown(), html: editor?.getHTML() })
-        }}
-        disableLocalStorage
-      />
+    <div className='rounded-lg border py-2 text-center'>
+      <button onClick={() => setOpenModal(() => true)} className='w-full'>
```
[the remark this answers is not in the document]
Comment (devodii) [reply components/CustomInputs/TextEditorInput.tsx:12]: I tried this, it was accessible.
Another way would be to create a radio prop for full modal display.
This way, we must tweak the main TextEditor - Which isn't in the picture :(

File: components/CustomInputs/TextEditorInput.tsx [hunk 2]
```diff
@@ -1,18 +1,29 @@
 "use client"
 
+import { useState } from "react"
+import { Dialog, DialogContent } from "@shadcn/ui/dialog"
 import { Editor } from "novel"
 
 export function TextEditorInput({ value, onChange }) {
+  const [openModal, setOpenModal] = useState(false)
```
[the remark this answers is not in the document]
Comment (devodii) [reply components/CustomInputs/TextEditorInput.tsx:8]: Okay.
How should we do this?
Completely clueless :/

File: components/CustomInputs/TextEditorInput.tsx (hunk 1 above)
[the remark this answers is not in the document]
Comment (BleedingDev) [reply components/CustomInputs/TextEditorInput.tsx:12]: There are components for triggering the open. :)

[the remark this answers is not in the document]
Comment (devodii) [reply components/CustomInputs/TextEditorInput.tsx]: Fixed, thanks :)
I started with the default drag&open strategy, so i forgot we no longer needed that :)

**Review by BleedingDev — DISMISSED** [review]

**Pull request merged**